docker-local/wallet-events-producer-testing/app.js

```diff
@@ -0,0 +1,27 @@
+const { MongoClient } = require('mongodb');
+const fs = require('fs');
+const path = require('path');
+
+async function main() {
+    const uri = 'mongodb://mongodb:27017/?replicaSet=rs0';
+    const client = new MongoClient(uri);
+
+    try {
+        await client.connect();
+
+        const database = client.db('payment-wallet');
+        const collection = database.collection('payment-wallets-log-events');
```

Comment:
In this project we read data from **payment-wallets** collection

Reply:
i see this even on main branch, are you sure?
![image](https://github.com/user-attachments/assets/c8c85680-dd7f-4621-bf64-394a910dca84)

Reply:
DB Name: payment-wallet
Collection name: payment-wallets

Reply:
done here 8afa8fc4433641cde88a4495e2b402b0e2202b30